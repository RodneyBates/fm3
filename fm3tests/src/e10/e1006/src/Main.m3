MODULE Main

EXPORTS Main
, Exp  

; FROM Exp IMPORT E1
; FROM Exp IMPORT
    E2
  , E3
  , E4

; FROM Exp IMPORT D1
; FROM Exp IMPORT M3 
; FROM Exp IMPORT D2 

; IMPORT Exp
; IMPORT Exp 
; BEGIN
  END Main
.

