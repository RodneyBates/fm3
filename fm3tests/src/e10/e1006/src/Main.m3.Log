[31mERROR: [0mMain.m3[10,18] Duplicate import of "E1", previously introduced at [8,2],
           original declaration at Exp.i3:[7,8], (2.5.1)
[31mERROR: [0mMain.m3[12,4] Duplicate import of "E2", previously introduced at [8,2],
           original declaration at Exp.i3:[8,8], (2.5.1)
[31mERROR: [0mMain.m3[13,4] Duplicate import of "E3", previously introduced at [8,2],
           original declaration at Exp.i3:[9,8], (2.5.1)
[31mERROR: [0mMain.m3[14,4] Duplicate import of "E4", previously introduced at [8,2],
           original declaration at Exp.i3:[10,8], (2.5.1)
[31mERROR: [0mMain.m3[16,18] Interface Exp has no declaration named "D1"
       (It's there via export/import, but not transitively importable (2.5.1).)
[31mERROR: [0mMain.m3[17,18] Interface Exp has no declaration named "M3"
[31mERROR: [0mMain.m3[18,18] Interface Exp has no declaration named "D2"
[31mERROR: [0mMain.m3[21,9] Duplicate import of "Exp", previously introduced at [20,9],
           original declaration at Exp.i3:[1,10], (2.5.1)
