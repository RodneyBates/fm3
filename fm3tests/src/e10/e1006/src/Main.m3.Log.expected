[31mERROR: [0mMain.m3[4,3] Duplicate export of "M4", previously from Main.m3:(3,8),
       declared at Main.i3:(6,8), (2.5.1)
[31mERROR: [0mMain.m3[6,18] Duplicate import of "E1", previously from Main.m3:(4,3),
       declared at Exp.i3:(7,8), (2.5.1)
[31mERROR: [0mMain.m3[8,4] Duplicate import of "E2", previously from Main.m3:(4,3),
       declared at Exp.i3:(8,8), (2.5.1)
[31mERROR: [0mMain.m3[9,4] Duplicate import of "E3", previously from Main.m3:(4,3),
       declared at Exp.i3:(9,8), (2.5.1)
[31mERROR: [0mMain.m3[10,4] Duplicate import of "E4", previously from Main.m3:(4,3),
       declared at Exp.i3:(10,8), (2.5.1)
[31mERROR: [0mMain.m3[12,18] Interface Exp has no declaration named "D1"
       (It's there via export/import, but not transitively importable (2.5.1).)
[31mERROR: [0mMain.m3[13,18] Interface Exp has no declaration named "M3"
[31mERROR: [0mMain.m3[14,18] Interface Exp has no declaration named "D2"
[31mERROR: [0mMain.m3[17,9] Duplicate import of "Exp", previously from Main.m3:(16,9),
       declared at Exp.i3:(1,10), (2.5.1)
